pages/common/opt.md

```diff
@@ -0,0 +1,15 @@
+# opt
+
+> LLVM .bc -> .bc modular optimizer and analysis printer. `opt` takes LLVM source files as input, runs the specified optimizations or analyses on it, and then outputs the optimized file or the analysis results.
```

Comment:
This seems rather wordy. Can we make it more concise at all?

Also, what is meant by `.bc`?

Reply:
> This seems rather wordy. Can we make it more concise at all?

Ok. I will think on something else.

> Also, what is meant by .bc?

`.bc` is a bitcode file while `.ll` is a human-readable version of it.

Reply:
Yea, this is a bit too long for a tldr page. How about - "A tool that takes LLVM source files and runs specified optimizations or analyses on them" ?